fun/cms/utils.py

```diff
@@ -0,0 +1,7 @@
+from unittest import skipUnless
+
+from django.conf import settings
+
+def skipUnlessCms(func):
+    skipUnless(settings.ROOT_URLCONF == 'fun.cms.urls', 'Test only valid in cms')(func)
+    return func
```

Comment:
hmmm ce n'est pas plutôt : 

        return skipUnless(settings.ROOT_URLCONF == 'fun.cms.urls', 'Test only valid in cms')(func)

?